package.json

```
@@ -26,7 +26,7 @@
     "lodash.defaultsdeep": "^4.6.1"
   },
   "engines": {
-    "node": "8.* || >= 10.*"
+    "node": "10.* || >= 12.*"
```

Lets go with:

```suggestion
    "node": "10.* || 12.* || >= 14"
```

Node 13 is already EOL'ed